wallet/reference/provider-api.md

```diff
@@ -256,6 +254,58 @@ For example, if you create a subscription using
 [`eth_subscribe`](/wallet/reference/eth_subscribe), each
 subscription update is emitted as a `message` event with a `type` of `eth_subscription`.
 
+## Methods
+
+### `removeListeners`
+
+Use the `removeListener` method to remove specific event listeners from an `EventEmitter` object. 
+In the following example `removeListener` is used to remove the `connect` and `accountsChanged` events:
+
+```javascript
+window.ethereum.on('_initialized', updateWalletAndAccounts);
```

Comment:
```suggestion
provider // Or window.ethereum if you don't support EIP-6963.
  .on("_initialized", updateWalletAndAccounts);
```

Also, maybe we should update all `window.ethereum` instances to support EIP-6963 by default similar to the rest of the code samples.

Reply:
Ok. Do you think tabs would work well for this? Looks like a lot with the comments.

```
provider // Or window.ethereum if you don't support EIP-6963
  .on("_initialized", updateWalletAndAccounts);
provider // Or window.ethereum if you don't support EIP-6963
  .on("connect", updateWalletAndAccounts);
provider // Or window.ethereum if you don't support EIP-6963
  .on("accountsChanged", updateWallet);
provider // Or window.ethereum if you don't support EIP-6963
  .on("chainChanged", updateWalletAndAccounts);
provider // Or window.ethereum if you don't support EIP-6963
  .on("disconnect", disconnectWallet);

return () => {
  provider // Or window.ethereum if you don't support EIP-6963
    .removeListener("connect", updateWalletAndAccounts);
  provider // Or window.ethereum if you don't support EIP-6963
    .removeListener("accountsChanged", updateWallet);
```

Reply:
Probably one comment at the top would suffice.